diagnostic.md

```diff
@@ -6,49 +6,56 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The back end is non user facing data and processes to support what goes on on the front end.
```

Comment:
not wrong, but you didn't really describe what a backend was, just what it wasn't and what it helped.